src/main/java/org/apache/commons/lang3/text/translate/CharSequenceTranslator.java

```diff
@@ -79,6 +81,10 @@ public final String translate(final CharSequence input) {
      * @param out Writer to translate the text to
      * @throws IOException if and only if the Writer produces an IOException
      */
+    @SuppressWarnings("argument.type.incompatible") /*
+    #1, #2 - argument to write() need not be @NonNegative
```

Comment:
If that's so, do you want to make a PR against the annotated JDK to fix the annotation there?